### ISSUE TITLE
[BUG] Starter Blocks and Ships and Weapons Spawning in overheated

### ISSUE THREAD

**Issue by Kadzar** [tsunamayo/Starship-EVO#2626]
### Short Description of the issue:

While playing, I noticed starter blocks and ships and weapons would spawn in overheated. I had spawned a ship earlier in the session with no trouble.

### Steps to reproduce the issue:

I noticed it when the weapon I was holding wouldn't fire and had no stats, so I got out a starter block to spawn in a new one, and I noticed it was overheated. Also, before this, I had used unrestricted teleporting.
### Environment
- Starship EVO version: 20w27d
- Graphics Card: NVIDIA GeForce GTX 1080

### Additional information:

![20200704132931_1](https://user-images.githubusercontent.com/66035211/86518918-f8b05200-bdfa-11ea-9e69-32ad8ec2b236.jpg)
![20200704132924_1](https://user-images.githubusercontent.com/66035211/86518922-fea63300-bdfa-11ea-975f-17852b7b0575.jpg)

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4873621/output_log.txt)

**Comment by tsunamayo** [comment]
Yes I need to handle better these case, and kill the player. Thanks
